content/applications/inventory_and_mrp/quality/quality_management/quality_alerts.rst

```diff
@@ -1,81 +1,91 @@
-- :guilabel:`Product Variant`: the specific variant of the product that has the quality issue, if
-  applicable
-- :guilabel:`Lot`: the lot number assigned to the product
-- :guilabel:`Work Center`: the work center where the quality issue originated
-- :guilabel:`Picking`: the picking operation during which the quality issue originated
-- :guilabel:`Team`: the quality team that will be notified by the quality alert
-- :guilabel:`Responsible`: the individual responsible for managing the quality alert
-- :guilabel:`Tags`: classify the quality alert based on user-created tags
-- :guilabel:`Root Cause`: the cause of the quality issue, if known
-- :guilabel:`Priority`: assign a priority between one and three stars to ensure more
-  urgent issues are prioritized
-
-The tabs at the bottom of the form can be used to provide additional information to quality teams:
-
-- :guilabel:`Description`: provide additional details about the quality issue
-- :guilabel:`Corrective Actions`: the method for fixing affected products
-- :guilabel:`Preventive Actions`: procedures for preventing the issue from occurring in
-  the future
-- :guilabel:`Miscellaneous`: the product vendor (if applicable), the company that produces
-  the product, and the date assigned
-
-.. image:: quality_alerts/quality-alert-form.png
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+
+In the Odoo *Quality* app, *quality alerts* are used to notify quality teams of product defects or
+other issues. Quality alerts can be created from a manufacturing or inventory order, from a work
+order in the *Shop Floor* module, or directly within the *Quality* app.
+
+Create quality alert
+====================
+
+There are four ways to create a new quality alert:
+
+- Navigate to :menuselection:`Quality -->  Quality Control --> Quality Alerts`, and then click
+  :guilabel:`New` to open a quality alert form.
+- Navigate to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and then
+  select an |MO|. Click the :guilabel:`Quality Alert` button at the top of the |MO| to open a
+  quality alert form in a new page. This method can only be used if a quality check has been
+  requested for the |MO|. The :guilabel:`Quality Alert` button will not appear otherwise.
+- Open the :menuselection:`Inventory` app, click the :guilabel:`# To Process` button on an inventory
+  order type card (Receipts, Delivery Orders, etc.), and then select an order. Click the
+  :guilabel:`Quality Alert` button at the top of the order to open a quality alert form in a new
+  page. This method can only be used if a quality check has been requested for the inventory order.
+  The :guilabel:`Quality Alert` button will not appear otherwise. If the button does not appear, a
+  quality alert can also be created by clicking the :guilabel:`⚙️ (gear)` icon at the top of the
+  page and selecting the :guilabel:`Quality Alert` option from the resulting menu.
```

Comment:
```suggestion
- Open the :menuselection:`Inventory` app, click the :guilabel:`# To Process` button on an inventory
  order type card (:guilabel:`Receipts`, :guilabel:`Delivery Orders`, etc.), and then select an
  order. Click the :guilabel:`Quality Alert` button at the top of the order to open a quality alert
  form in a new page.
  
.. important::  
   This method can **only** be used if a quality check has been requested for the inventory order.
   The :guilabel:`Quality Alert` button will **not** appear otherwise.
   
   If the button does not appear, a quality alert can also be created by clicking the :guilabel:`⚙️
   (gear)` icon at the top of the page, and selecting the :guilabel:`Quality Alert` option from the
   resulting menu.
```
added another important admonition for the same reasons mentioned above.

Also, when I attempted to find the Quality Alert options for the gear icon, no such option was available on the 16.4 runbot I used.

And I made sure that Quality Alerts was activated in the MRP app, and still couldn't find it.

I opened Inventory, clicked # to Process on Receipts (per the instructions) and saw the following:
https://drive.google.com/file/d/1ik1_RQDybLbhw1AJmxYrwk9m2LPLtAjm/view?usp=sharing

Please double-check the validity of this. Thanks! 👍